lupa/_lupa.pyx

```diff
@@ -1267,8 +1267,14 @@ cdef object execute_lua_call(LuaRuntime runtime, lua_State *L, Py_ssize_t nargs)
     # call into Lua
     with nogil:
         result_status = lua.lua_pcall(L, nargs, lua.LUA_MULTRET, 0)
-    #runtime.reraise_on_exception()
     if result_status:
+        python_errors = (
+            'error reading Python attribute/item',
+            'error during Python call',
+        )
```

Comment:
Yeah, I thought about it, but the issue is that I have to know if the error is Python related or Lua-related, and thus I have to distinguish those errors somehow. The only way I can suggest instead is to use some object instead of string in luaL_error with some code attribute that would signal that this is a python-related error (can we actually use python exception there as it's a usual python object actually? then we would not even need to store the last catched python exception).

But it was tricky for me to decide to use it because it's a major change and may affect error handling by tests/other projects depending on lupa.